lib/ood_core/job/adapters/linux_host/launcher.rb

```diff
@@ -116,27 +116,27 @@ def call(cmd, *args, env: {}, stdin: "")
     s.success? ? o : raise(Error, e)
   end
 
-  # The SSH invocation to send a command
+  # The SSH invocation to send the command you pass into cmd (Array<String>)
```

Comment:
I had to read this sentence several times before I realized that "command you pass into cmd (Array<String>)" is referring to the positional argument `cmd`.

It might preferable to add the yardoc description for the positional arguments here, below the description i.e.

    @param destination_host ...
    @param cmd ...